components/tidb_query_vec_executors/src/index_scan_executor.rs

```diff
@@ -193,7 +236,7 @@ impl ScanExecutorImpl for IndexScanExecutorImpl {
                     EvalType::Int,
                 ));
             }
-            DecodeCommonHandle => {
+            DecodeCommonHandle | DecodeCommonHandleRowKey => {
```

Comment:
It is a bad idea to mix DAG scanners and Analyze scanners..

Reply:
If create a new Analyze scanner, would there be more duplicated code?